dist/jakecache.js

```diff
@@ -65,14 +65,18 @@ const _status = Symbol('status')
 
 class JakeCache extends PolyfilledEventTarget {
   constructor () {
-    super(['bort', 'cached', 'checking',
+    super(['abort', 'cached', 'checking',
            'downloading', 'error', 'obsolete',
            'progress', 'updateready', 'noupdate'])
 
     if (window.jakeCache) {
       return window.jakeCache
     }
     window.jakeCache = this
+    
+    if (('serviceWorker' in navigator) === false) {
```

Comment:
Maybe we should throw some error here?

Reply:
Is this really an error or just regulary aborts the assignment of the serviceworker eventHandler for incompliant Browser for using the AppCache  fallback?

Reply:
Well the polyfill won't work then, and you need to rely on native appcache support. I guess it is better to fail, because people should probably do such a test themselves before using the polyfill.

Question is which browsers won't have either. I guess the test could also be whether it is http or https, as service worker only works on https, and now Mozilla is moving appcache to https as well.

Some more thought about this and description in the readme could be useful

Reply:
ok i need to dig a bit deeper into the jakecache polyfill concept...